cli/src/utils/user-config.ts

```diff
@@ -0,0 +1,13 @@
+import { join } from 'path';
+import { homedir } from 'os';
+import Conf from 'conf';
+import { safeDump, safeLoad } from 'js-yaml';
+
+export default new Conf({
+  fileExtension: 'yml',
+  serialize: safeDump,
+  deserialize: safeLoad,
+  clearInvalidConfig: false,
+  configName: 'shiftjs.config',
```

Comment:
Just call this `config` if it's already in `~/.shiftjs`.

Reply:
from the spec:
> When the user successfully completes the process, logic will create a credentials file at `~/.shiftjs/shiftjs.config.yml` which will store the authenticated credentials.